tests/test_routes.py

```diff
@@ -5,38 +5,49 @@
   nosetests -v --with-spec --spec-color
   coverage report -m
 """
+from crypt import methods
 import os
 import logging
+import json
+import unittest
 from unittest import TestCase
 from unittest.mock import MagicMock, patch
+from service.utils import status  # HTTP Status Codes
+from service.models import db, init_db, Product
 from service import app
-from service.models import db
-from service.common import status  # HTTP Status Codes
+from tests.factories import ProductFactory
 
 
```

Comment:
according to the professor's GitHub file maybe we should add a DATABASE_URI here:
the detail we would discuss in tomorrow
otherwise, we could not know where the DATABASE_URI in line32 comes from

eg:
/# DATABASE_URI = os.getenv('DATABASE_URI', 'sqlite:///../db/test.db')
DATABASE_URI = os.getenv(
    "DATABASE_URI", "postgresql://postgres:postgres@localhost:5432/testdb"
)
BASE_URL = "/pets"